builder.config.json

```diff
@@ -39,15 +39,15 @@
           "name": "home",
           "path": "\/",
           "methods": ["GET"]
-        }
-      },
-      "trees": {
+        },
         "editTree": {
           "name": "editTree",
-          "path": "\/trees\/{treeId}\/{component?}\/{mode?}",
-          "params": ["treeId", "component", "mode"],
+          "path": "\/flows\/{flowId}\/{component?}\/{mode?}",
```

Comment:
Do you still need `{component?}` here given this is now static in the router? I might be misunderstanding how this fits together.

Reply:
I am guessing we need the `{component?}` to switch between flow and resource views. Do you think we might need something new instead of changing the component?

Reply:
Okay, so this intended for a future feature? I think this is fine to leave for now.